model/map-jpa/src/main/java/org/keycloak/models/map/storage/jpa/hibernate/contributor/JpaMapFunctionContributor.java

```diff
@@ -53,9 +53,8 @@ private BasicType getBasicType(FunctionContributions fc, BasicTypeReference<?> b
         return fc.getTypeConfiguration().getBasicTypeRegistry().resolve(btr);
     }
 
-    // Since Hibernate 6.2.0 dialect should be accessible via FunctionContributions.getDialect(): https://github.com/hibernate/hibernate-orm/pull/5964/
     private Dialect getDialect(FunctionContributions fc) {
```

Comment:
At this point, why even have this method? Might as well call it directly on `fc` wherever it's used.